app/src/test/java/org/mozilla/tv/firefox/framework/FrameworkRepoTest.kt

```diff
@@ -7,59 +7,96 @@ package org.mozilla.tv.firefox.framework
-    fun `WHEN init is called THEN a touch exploration state change listener is added`() {
-        val a11yManager = mock(AccessibilityManager::class.java)
-        repo.init(a11yManager)
-        verify(a11yManager, times(1)).addTouchExplorationStateChangeListener(any(TouchExplorationStateChangeListener::class.java))
+    fun `GIVEN the framework has voice view disabled WHEN init is called THEN the voice view is disabled`() {
+        val accessibilityManager = touchExplorationA11yManagerWrapper.mock.also {
+            `when`(it.isTouchExplorationEnabled).thenReturn(false)
+        }
+
+        repo.isVoiceViewEnabled.assertValues(false) {
+            repo.init(accessibilityManager)
+        }
     }
 
     @Test
-    fun `WHEN init is called THEN an initial touch exploration state is set from the accessibility manager`() {
-        val a11yManager = mock(AccessibilityManager::class.java)
-        `when`(a11yManager.isTouchExplorationEnabled).thenReturn(true)
-        repo.init(a11yManager)
-        assertTrue(repo.isVoiceViewEnabled.value!!)
-
-        `when`(a11yManager.isTouchExplorationEnabled).thenReturn(false)
-        repo.init(a11yManager)
-        assertFalse(repo.isVoiceViewEnabled.value!!)
+    fun `GIVEN the framework has voice view enabled WHEN init is called THEN the voice view is enabled`() {
+        val accessibilityManager = touchExplorationA11yManagerWrapper.mock.also {
+            `when`(it.isTouchExplorationEnabled).thenReturn(true)
+        }
+
+        repo.isVoiceViewEnabled.assertValues(true) {
+            repo.init(accessibilityManager)
+        }
     }
 
     @Test
-    fun `WHEN touch exploration state changes to false THEN voice view is disabled`() {
-        repo.touchExplorationStateChangeListener.onTouchExplorationStateChanged(false)
-        repo.isVoiceViewEnabled.observeForever { assertFalse(it!!) }
+    fun `GIVEN init is called WHEN the framework emits touch exploration disabled THEN voice view is disabled`() {
+        repo.init(touchExplorationA11yManagerWrapper.mock)
+
+        val defaultValue = touchExplorationA11yManagerWrapper.isTouchExplorationStateEnabled
+        repo.isVoiceViewEnabled.assertValues(defaultValue, false) {
+            touchExplorationA11yManagerWrapper.isTouchExplorationStateEnabled = false
+        }
     }
 
     @Test
-    fun `WHEN touch exploration state changes to true THEN voice view is enabled`() {
-        repo.touchExplorationStateChangeListener.onTouchExplorationStateChanged(true)
-        repo.isVoiceViewEnabled.observeForever { assertTrue(it!!) }
+    fun `GIVEN init is called WHEN the framework emits touch exploration enabled THEN voice view is enabled`() {
+        repo.init(touchExplorationA11yManagerWrapper.mock)
+
+        val defaultValue = touchExplorationA11yManagerWrapper.isTouchExplorationStateEnabled
+        repo.isVoiceViewEnabled.assertValues(defaultValue, true) {
+            touchExplorationA11yManagerWrapper.isTouchExplorationStateEnabled = true
```

Comment:
Nit: it would be good to then push and check for false.  This would avoid false positives in the case where default == true.

Reply:
Addressed and confirmed fix on slack.